devices/RgbDiode/README.md

```diff
@@ -0,0 +1,28 @@
+# RGB diode - PWM
+Library designed for controlling RGB diodes using PWM.
```

Comment:
Some Markdown 101: you always need an empty line after a title and an empty line before except for main title

```suggestion

Library designed for controlling RGB diodes using PWM.
```